modules/charon-core/src/main/java/org/wso2/charon3/core/objects/Group.java

```diff
@@ -88,7 +88,7 @@ public void setDisplayName(String displayName) throws CharonException, BadReques
             }
             return memberList;
         } else {
-            return null;
+            return Collections.emptyList();
```

Comment:
Let's return the empty array list as it returns an immutable list.

Reply:
refer to this https://github.com/wso2-support/charon/pull/38